frappe/erpnext
# fix: Taxes and Charges template not getting copied from Purchase Order/Receipt to Invoice

On creating a Purchase Invoice from Purchase Order, the taxes and charges template from Purchase Receipt was not getting copied to Purchase Invoice.

Steps to test:

1. Create a purchase receipt and select a Purchase Taxes and Charges Template and save and submit the PR
<img width="1336" alt="Screenshot 2021-11-30 at 8 54 03 PM" src="https://user-images.githubusercontent.com/42651287/144076128-e0c1b252-c967-4731-92b8-5c384d3649c5.png">

2. Create a Purchase Invoice from the PR
<img width="784" alt="Screenshot 2021-11-30 at 8 56 50 PM" src="https://user-images.githubusercontent.com/42651287/144076315-01d7390a-8d5d-4e16-94e1-5d26e1dffa5a.png">

3. Purchase Taxes and Charges template should be copied from Purchase Receipt to Invoice
![image](https://user-images.githubusercontent.com/42651287/144077029-eb0b5960-f581-4457-9fb9-8070cef638c3.png)
